spec/features/admin/delete_organisation_as_super_admin_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'features/support/sign_up_helpers'
+
+describe 'deleting an organisation' do
+  let!(:admin_user) { create(:user, super_admin: true) }
```

Comment:
nit: Don't think you need the `!` here

```suggestion
  let(:admin_user) { create(:user, super_admin: true) }
```